BUILD.bazel

```diff
@@ -1884,15 +1884,17 @@ cc_test(
 # jit tests
 # Because these individual unit tests require custom registering,
 # it is easier to mimic the cmake build by globing together a single test.
+jit_test_source = glob([
+  "test/cpp/jit/*.cpp",
+  "test/cpp/jit/*.h",
+  "test/cpp/tensorexpr/*.cpp",
+  "test/cpp/tensorexpr/*.h",
+])
+jit_test_source.remove("test/cpp/jit/test_flatbuffer.cpp")
```

Comment:
Can you please add a comment, why this test should not be run in OSS CI?

Reply:
Because this PR's goal is to add flatbuffer to CMake's target only. Previous attempts add flatbuffer to cmake/bazel/fbcode's TARGET/xplat's BUCK at the same time and had bigger impact surfaces. So we opted to add those in separate PR's so each can be reverted individually if something unexpected happens.

Reply:
This sounds wrong. You can make this an internal only feature initially, but I would prefer no feature discrepancy between CMAKE and Bazel